package.json

```diff
@@ -5,9 +5,12 @@
     "packages/*"
   ],
   "scripts": {
-    "dev": "nodemon",
-    "build": "yarn workspace veto-backend build && yarn workspace veto-frontend build && yarn workspace screen-app build",
+    "dev": "yarn cyphernode:start && yarn start",
     "start": "NODE_ENV=${NODE_ENV:-dev} concurrently --kill-others-on-fail 'yarn start:backend' 'yarn start:frontend'",
+    "cyphernode:start": "SKIP_VETO=true ./start.sh",
+    "cyphernode:stop": "./stop.sh",
+    "cyphernode:setup": "cd cyphernode && CFG_PASSWORD=veriphirocks! ./setup.sh -irc && cd ..",
```

Comment:
we could also use `SKIP_VETO=true ./build.sh`

Now that I think of it, maybe we should rename `build.sh` to `setup.sh` for consistency?